src/services/webhook.ts

```
@@ -21,27 +29,49 @@ export class WebhookService {
-        return { 
-          message: 'Analysis triggered', 
-          prNumber: pr.number,
-          shouldAnalyze: true 
+
+        const analysisMessage: AnalysisMessage = {
+          repository: payload.repository.full_name,
+          owner: payload.repository.owner.login,
+          repo: payload.repository.name,
+          pullRequestNumber: pr.number,
+          action
         };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)